rust/azure_iot_operations_services/examples/lock_client.rs

```diff
@@ -194,6 +195,7 @@ async fn lock_client_2_operations(
     let shared_resource_key_set_options = SetOptions {
         set_condition: SetCondition::Unconditional,
         expires: Some(Duration::from_secs(15)),
+        ..Default::default()
```

Comment:
this would be relevant the same way for lease/lock, right? There isn't any reason that we'd obfuscate this at this layer?